### PR TITLE
feat(clerk-js): Use borders instead of box shadow borders

### PR DESCRIPTION
**Pull request by desiprisg**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by desiprisg** [comment]
!preview

File: packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ const FabContent = ({ title, signOutText }: FabContentProps) => {
         localizationKey={signOutText}
         onClick={async () => {
           // clerk-js has been loaded at this point so we can safely access session
-          await signOut({ sessionId: session!.id });
+          await signOut({ sessionId: session.id });
```
Comment (octoper) [inline packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:93]: Is this change intentional?

Comment (anagstef) [reply packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:93]: I think linter removes this when you make changes to the file, and then the tests break 😢

Comment (desiprisg) [reply packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx]: It's the linter yes. I reverted.

**Comment by anagstef** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @anagstef, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-pk4u8un7w.clerkstage.dev) | Jan 10, 2024 04:21 PM |

**Comment by desiprisg** [comment]
!preview

**Comment by octoper** [comment]
!preview

**Comment by desiprisg** [comment]
!preview

**Review by octoper — APPROVED** [review]

**Review by anagstef — APPROVED** [review]
